packages/esbuild-plugin-import-path-remapper/test/esbuild-plugin-import-path-remapper.test.js

```diff
@@ -0,0 +1,112 @@
+// @ts-check
+"use strict";
+
+describe("@rnx-kit/esbuild-plugin-import-path-remapper", () => {
+  const esbuild = require("esbuild");
+  const ImportPathRemapperPlugin = require("../lib/index");
```

Comment:
You should be able to consume TypeScript with `@rnx-kit/jest-preset`.
```suggestion
  const ImportPathRemapperPlugin = require("../src/index");
```